Examples/Tests/initial_distribution/analysis_distribution.py

```diff
@@ -0,0 +1,132 @@
+#! /usr/bin/env python
+
+# Copyright 2019-2020 Yinjian Zhao
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+# This script tests initial distributions.
+# 1 denotes gaussian distribution.
+# 2 denotes maxwell-boltzmann distribution.
+# 3 denotes maxwell-juttner distribution.
+# 4 denotes gaussian position distribution.
+# The distribution is obtained through reduced diagnostic ParticleHistogram.
+
+import numpy as np
+import scipy.constants as scc
+import scipy.special as scs
+
+# print tolerance
+tolerance = 0.009
+print('tolerance:', tolerance)
+
+#===============================
+# gaussian and maxwell-boltzmann
+#===============================
+
+# load data
+h1x = np.genfromtxt("h1x.txt")
+h1y = np.genfromtxt("h1y.txt")
+h1z = np.genfromtxt("h1z.txt")
+h2x = np.genfromtxt("h2x.txt")
+h2y = np.genfromtxt("h2y.txt")
+h2z = np.genfromtxt("h2z.txt")
+
+# parameters of bin
+bin_min = -4.0e-2 * scc.c
+bin_max = +4.0e-2 * scc.c
+bin_num = 50
+bin_size = (bin_max-bin_min)/bin_num
+
+# parameters of theory
+u_rms = 0.01
+gamma = np.sqrt(1+u_rms*u_rms)
+v_rms = u_rms / gamma * scc.c
+
+# compute the analytical solution
+f = np.zeros( bin_num )
+for i in range(bin_num):
+    v = (i+0.5)*bin_size + bin_min
+    f[i] = np.exp(-0.5*(v/v_rms)**2)/(v_rms*np.sqrt(2.0*scc.pi))
+
+# normalization
+f = f / np.amax( f )
```

Comment:
Could you read the bins from the output file? I find that re-defining the bins is **extremely** error-prone, so it would be nice to have a nice example of how to use this diagnostics here.

An example of how it could look (but that's not tested, though) is shown below, hopefully it should be readable and safe.

```suggestion
vbins = read_reduced_diags_histogram("h1x.txt")[2]
u_rms = 0.01
v_rms = u_rms / np.sqrt(1+u_rms*u_rms) * scc.c
f = np.exp(-0.5*(vbins/v_rms)**2)/(v_rms*np.sqrt(2.0*scc.pi))
f /= np.amax( f )
```
Of course this also applies below.